packages/notebook-extension/src/index.ts

```diff
@@ -2319,9 +2323,8 @@ function addCommands(
         if (wideget) {
           if (Private.renderSideBySide) {
             return NotebookActions.renderSideBySide(wideget.content);
-          } else {
-            return NotebookActions.renderNotSideBySide(wideget.content);
           }
+          return NotebookActions.renderNotSideBySide(wideget.content);
```

Comment:
Mind taking the opportunity to fix the small typo?

```suggestion
          return NotebookActions.renderNotSideBySide(widget.content);
```

Thanks!

Reply:
😀 Typo fixed

Reply:
Thanks!